### PR TITLE
Add a bit more detail to our README

### PR DESCRIPTION
**Pull request by lleyton**
Currently it's a bit barren, but I'll update it once I'm done writing our policies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
-# Terra repositories
+# Terra Sources
 
-This monorepo contains the package manifests for the Terra repositories.
+Terra is a rolling-release Fedora repository for all the software you need.
+With Terra, you can install the latest packages knowing that quality and security are assured.
+Contributions are always welcome, but please read our [documentation](https://github.com/terrapkg/packages/wiki) first.
```
Comment (windowsboy111) [inline README.md:5]:
```suggestion
[Contributions](../wiiki/Contribute) are always welcome, but please read our [documentation](../wiki) first.
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 
+This monorepo contains the package manifests for the Terra repositories.
```
Comment (windowsboy111) [inline README.md:7]:
```suggestion
This monorepo contains the package manifests for all packages in Terra.
```

File: README.md [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
-# How to use
+## Installation
 ```bash
 sudo dnf config-manager --add-repo https://github.com/terrapkg/subatomic-repos/raw/main/terra.repo
 ```
-# Add packages
-Pull requests are welcomed! See [here](docs/CREATING.md) for instructions.
+
+## Documentation
+Our documentation is located on our [GitHub Wiki](https://github.com/terrapkg/packages/wiki).
```
Comment (windowsboy111) [inline README.md:15]:
```suggestion
Our documentation can be found on our [GitHub Wiki](https://github.com/terrapkg/packages/wiki).
```

File: README.md [hunk 4]
```diff
@@ -1,11 +1,19 @@
...
+
+## Questions?
+Feel free to reach out on [Discord](https://discord.gg/5fdPuxTg5Q), we're always happy to help!
```
Comment (windowsboy111) [inline README.md:18]:
```suggestion
Feel free to reach out on [Discord](https://discord.gg/5fdPuxTg5Q). We're always happy to help!
```

**Review by windowsboy111 — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,11 +1,19 @@
-# Terra repositories
+# Terra Sources
 
-This monorepo contains the package manifests for the Terra repositories.
+Terra is a rolling-release Fedora repository for all the software you need.
+With Terra, you can install the latest packages knowing that quality and security are assured.
+[Contributions](https://github.com/terrapkg/packages/wiiki/Contribute) are always welcome, but please read our [documentation](https://github.com/terrapkg/packages/wiki) first.
```
Comment (windowsboy111) [inline README.md:5]:
```suggestion
[Contributions](https://github.com/terrapkg/packages/wiki/Contribute) are always welcome, but please read our [documentation](https://github.com/terrapkg/packages/wiki) first.
```
My fault lmao

Comment (lleyton) [reply README.md]: haha

**Review by windowsboy111 — APPROVED** [review]

**Pull request merged**